roles/cinder-common/defaults/main.yml

```diff
@@ -56,6 +56,8 @@ cinder:
       - cinder-volume
       - cinder-volume-usage-audit
   heartbeat_timeout_threshold: 30
+  service:
```

Comment:
no, this is only for nova because cinder has python-barbicanclient in it's requirements.txt. Nova does not have it.